system/systemd.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2016, Brian Coca <bcoca@ansible.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+module: systemd
+author:
+    - "Ansible Core Team"
+version_added: "2.2"
+short_description:  Manage services.
+description:
+    - Controls systemd services on remote hosts.
+options:
+    name:
+        required: true
+        description:
+            - Name of the service.
+    state:
+        required: false
+        default: null
+        choices: [ 'started', 'stopped', 'restarted', 'reloaded' ]
+        description:
+            - C(started)/C(stopped) are idempotent actions that will not run commands unless necessary.
+              C(restarted) will always bounce the service. C(reloaded) will always reload.
+    enabled:
+        required: false
+        choices: [ "yes", "no" ]
+        default: null
+        description:
+            - Whether the service should start on boot. B(At least one of state and enabled are required.)
+    masked:
+        required: false
+        choices: [ "yes", "no" ]
+        default: null
+        description:
+            - Whether the unit should be masked or not, a masked unit is impossible to start.
+    daemon_reload:
```

Comment:
I don't think it merits a module as it would only ever do 'the one thing', it is here for the case in which the units have been modified (template/copy/etc tasks) and you want to execute the service action on the updated (reloaded) config and it seems to be a logical fit.